IdentityCore/src/cache/token/MSIDAppMetadataCacheItem.h

```diff
@@ -0,0 +1,22 @@
+//
+//  MSIDAppMetadataCacheItem.h
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 10/8/18.
+//  Copyright © 2018 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "MSIDJsonSerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDAppMetadataCacheItem : NSObject <NSCopying, MSIDJsonSerializable>
+
+@property (readwrite, nonnull) NSString *clientId;
```

Comment:
nit: readwrite is default, no need to add it here

Reply:
done